src/descriptors-device.adb

```diff
@@ -43,6 +43,27 @@ package body Descriptors.Device is
       Output_Dir : String);
    --  Dump the IRQ names and trap handlers
 
+   -----------------
+   -- MCU_To_CPU --
+   -----------------
+
+   function MCU_To_CPU
```

Comment:
You would need to declare the function first, at the beginning of the package. I'm a bit surprised this didn't raise a compilation error. I'll check the project file to make sure this rule is checked by gnat.

Reply:
We can put in place an automatic validation of the pull request with Travis-CI.